### DISCUSSION TITLE
Lora strength beyond -2 or +2

### DISCUSSION THREAD

**Discussion by Megaderp-del** [lllyasviel/Fooocus#3291] [Q&A] [answered]
Hi,

Is there a way to set the LorA strength to higher values? With the appearance of many sliders it would be awesome to have the possibility to set them to higher (or lower) values.

Regards :)

**Comment by mashb1t** [comment, marked as answer]
You can set the boundaries to whatever you want by setting them in the config.txt. Please check out the config example file for the exact key names.
